### ISSUE TITLE
"More information" link should lead to the info page, not the vsix download

### ISSUE THREAD

**Issue by Thieum** [madskristensen/ExtensionGallery#32]
![image](https://user-images.githubusercontent.com/944894/28531632-6d13b8bc-7065-11e7-9571-027d1b713aa3.png)

The more information link leads to the dowload of the extension instead of the page of the extension on vsix. (For example http://vsixgallery.com/extension/55d15546-28ca-40dc-af23-dfa503e9c5fe/ here)

**Comment by yannduran** [comment]
@Thieum If you haven't realised yet, the _More Information_ link that appears in the _Extensions and Updates_ dialog is populated by whatever you have entered in the _More Info URL_ field in the VSIX manifest editor (last field on the _Metadata_ tab) for your extension.

![image](https://user-images.githubusercontent.com/2031126/35184131-a6148858-fe45-11e7-9f03-07e6cfb799c8.png)

**Comment by Thieum** [comment]
@yannduran you're right, I'll open a PR on the projects that don't have the right url there.

**Comment by Thieum** [comment]
@yannduran ah no that's not right, the link actually works in the installed panel, but not in the online one and I guess it doesn't work either in the update one, for example:

![image](https://user-images.githubusercontent.com/944894/50931027-83dfa000-142f-11e9-9a75-c4fddb6fed6c.png)

even if the url is defined correctly:

![image](https://user-images.githubusercontent.com/944894/50931066-9a85f700-142f-11e9-9b7c-98f96e0abee2.png)

**Comment by Thieum** [comment]
would be fixed by #28 when deployed
